src/Functions/FunctionCustomWeekToDateOrDate32.h

```diff
@@ -20,9 +20,10 @@ class FunctionCustomWeekToDateOrDate32 : public IFunctionCustomWeek<Transform>,
         return std::make_shared<FunctionCustomWeekToDateOrDate32>(context_);
     }
 
-    explicit FunctionCustomWeekToDateOrDate32(ContextPtr context_) : WithContext(context_)
+    explicit FunctionCustomWeekToDateOrDate32(ContextPtr context_)
```

Comment:
Minor: + `const` in l. 16 (fine to do in some follow-up PR)

Reply:
I will correct it now

Reply:
Done